<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="first_page">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000000</width>
        <height>1000000</height>
       </size>
      </property>
      <attribute name="title">
       <string>刷视频助手</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0">
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>1920</width>
           <height>1080</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
            <item>
             <widget class="QGroupBox" name="setting_groupBox">
              <property name="title">
               <string>设置栏</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="setting_layout" stretch="0,1">
                 <item>
                  <layout class="QVBoxLayout" name="left_layout" stretch="0,0,0,0,0,0,1">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>波特率：</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="baudrate_select">
                       <property name="currentText">
                        <string>9600</string>
                       </property>
                       <property name="currentIndex">
                        <number>4</number>
                       </property>
                       <item>
                        <property name="text">
                         <string>600</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>1200</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>2400</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>4800</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>9600</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>14400</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>19200</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>28800</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>38400</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>串口名：</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="serial_name">
                       <property name="text">
                        <string>COM3</string>
                       </property>
                       <property name="placeholderText">
                        <string>请输入串口名</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>延时：</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>12</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QComboBox" name="delay_time">
                       <property name="currentIndex">
                        <number>1</number>
                       </property>
                       <item>
                        <property name="text">
                         <string>0.5s</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>1s</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>1.5s</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>2s</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QPushButton" name="open_serial_btn">
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="text">
                      <string>打开串口</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="close_serial_btn">
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="text">
                      <string>关闭串口</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="exit_app_btn">
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="text">
                      <string>退出程序</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Orientation::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="right_layout">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>cookie输入：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPlainTextEdit" name="cookie_text">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="plainText">
                      <string/>
                     </property>
                     <property name="placeholderText">
                      <string>请输入cookie</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QPushButton" name="add_cookie_btn">
                       <property name="cursor">
                        <cursorShape>PointingHandCursor</cursorShape>
                       </property>
                       <property name="text">
                        <string>添加cookie</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="open_bilibili_btn">
                       <property name="cursor">
                        <cursorShape>PointingHandCursor</cursorShape>
                       </property>
                       <property name="text">
                        <string>打开b站</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="log_groupBox">
              <property name="title">
               <string>日志栏</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QPushButton" name="clear_log_btn">
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="text">
                      <string>清空日志</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="save_log_btn">
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="text">
                      <string>保存日志</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Orientation::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPlainTextEdit" name="message_text">
                   <property name="cursor" stdset="0">
                    <cursorShape>ArrowCursor</cursorShape>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="second_page">
      <attribute name="title">
       <string>实时时钟</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="thrid_page">
      <attribute name="title">
       <string>温光检测</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
